### PR TITLE
Added Support for get data as Graphs and format response as graph

### PR DESCRIPTION
**Pull request by PowerShellToday**
#28 
Ok, so here it is.

### CODE REVIEW

File: PSNeo4j/Public/Invoke-Neo4jQuery.ps1 [hunk 1]
```diff
@@ -124,6 +128,12 @@
                 if($PSBoundParameters.ContainsKey('Parameters')) {
                     Add-Member -InputObject $Statement -Name 'parameters' -Value $Parameters -MemberType NoteProperty
                 }
+                if($AsGraph.IsPresent) {
+                    Add-Member -InputObject $Statement -Name 'resultDataContents' -Value ([string[]]('graph')) -MemberType NoteProperty
+                    if ($As -notin 'Raw','Graph') {
```
Comment (RamblingCookieMonster) [inline PSNeo4j/Public/Invoke-Neo4jQuery.ps1]: Hi!

I do like keeping the `AsGraph` switch (lets you use Raw and keep that data), but would it be worth doing something like this?

```powershell
if (-not $PSBoundParameters.ContainsKey(As)) {
    $As = 'Graph'
}
```

This way if the user doesn't explicitly set `As`, they can simply use `AsGraph` and get results that they would expect (a graph)?

If you'd rather not do that now, I can try adding it later, your call - Thanks!

Warren

Comment (PowerShellToday) [reply PSNeo4j/Public/Invoke-Neo4jQuery.ps1]: Sounds clever, I will get on it right now.

**Comment by PowerShellToday** [comment]
Maybe should add some tests as well?

**Comment by RamblingCookieMonster** [comment]
LGTM!  Sure, if you'd like to tackle any testing or an example in the readme that would be awesome!

**Comment by RamblingCookieMonster** [comment]
I could see adding a setting for `AsGraph` and the option for `-As Graph` to `Set-PSNeo4jConfiguration`, but definitely not enough to hold this up - merging!

**Pull request merged**